docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -87,6 +87,36 @@ You can also manually update your theme as shown in the snippet below:
 
 This reduces the API surface and lets you define variants in other slots of the component.
 
+## Accordion
```

Comment:
This page could become confusing. If we plan to continue to add these sections here, we should include the version in which the new API was introduced. Otherwise people may run the codemod while installing an older version (e.g. 6.0.0).

Reply:
I'll look into how to do this

Reply:
@mnajdova what do you think about this approach:

- For APIs introduced in 6.0.0, we don't add any callout
- For APIs introduced after 6.0.0, we add a warning callout specifying the version in which the API was added

If you agree, this PR doesn't need any change, as these APIs were introduced in 6.0.0. The deprecation messages were omitted while we discussed the direction of the slot pattern, which is why we are bringing them back now.

Reply:
Agree, ok I see, I thought the APIs were introduced later.